lib/challenge.js

```diff
@@ -114,7 +124,9 @@ function prompt6() {
  *   - I found a 10. High five!
  */
 function prompt7() {
-  // YOUR CODE HERE
+  for(let i = 0; i <= 100; i++) {
+    console.log('I found a' + i + '. High Five!')
+  }
 }
```

Comment:
This will print every instance into the console.  Challenge yourself to print only multiples of five (resubmission not required)
> Sample Output IN THE BROWSER:
> \- I found a 5. High five!
> \- I found a 10. High five!